psi4/src/psi4/ccresponse/optrot.cc

```diff
@@ -447,10 +449,21 @@ void optrot(std::shared_ptr<Molecule> molecule)
       outfile->Printf( "\n   Specific rotation using length-gauge electric-dipole Rosenfeld tensor.\n");
       outfile->Printf( "\t[alpha]_(%5.3f) = %10.5f deg/[dm (g/cm^3)]\n", params.omega[i], rotation_rl[i]);
 
-      if(params.wfn == "CC2")
-        Process::environment.globals["CC2 SPECIFIC ROTATION (LEN)"] = rotation_rl[i];
-      else if(params.wfn == "CCSD")
-        Process::environment.globals["CCSD SPECIFIC ROTATION (LEN)"] = rotation_rl[i];
+      /* grab omega in nm, rounded to nearest int */
+      long om_nm = std::lround((pc_c*pc_h*1e9)/(pc_hartree2J*params.omega[i]));
+
+      if(params.wfn == "CC2"){
+        /* build up a string */
```

Comment:
keep `/* grab omega in nm, rounded to nearest int */`, drop the two `/* build up a string */`

same below for LEN & MVG

Reply:
Done